src/app/components/Panel/Sort/Sort.js

```diff
@@ -19,8 +25,10 @@ export const Sort = () => {
       <div className={clsx(styles.dropdown, { [styles.isOpened]: isOpened })}>
         <Collapse isOpened={isOpened}>
           <ul>
-            <li>Rating Lowest</li>
-            <li>Rating Highest</li>
+            <li onClick={() => handleClick("metacritic")}>Rating Lowest</li>
+            <li onClick={() => handleClick("-metacritic")}>Rating Highest</li>
+            <li onClick={() => handleClick("-released")}>Newest</li>
+            <li onClick={() => handleClick("released")}>Oldest</li>
```

Comment:
This could be put into some sort of config, then mapped. Would then look way cleaner and wouldn't violate the DRY principle.